exatomic/nwchem/output.py

```diff
@@ -257,6 +264,167 @@ def parse_basis_set_order(self):
         bso['shell'] = shls
         self.basis_set_order = bso
 
+    def parse_roa(self):
+        """
+        Parse the :class:`~exatomic.core.tensor.Polarizability` dataframe. This will parse the
+        output from the Raman Optical Activity outputs.
+
+        Note:
+            We generate a 3D tensor with the 2D tensor code. 3D tensors will have 3 rows labeled
+            with the same name.
+        """
+        _reroa = 'roa begin'
+        _reare = 'alpha real'
+        _reaim = 'alpha im'
+        _reombre = 'omega beta(real)'
+        _reombim = 'omega beta(imag)'
+        _redqre = 'dipole-quadrupole real (Cartesian)'
+        _redqim = 'dipole-quadrupole imag (Cartesian)'
+
+        if not self.find(_reroa):
+            return
```

Comment:
You bring up a good point about informing the user about the behavior of the code.  I think adopting more widespread logging into the source would allow for a configurable solution to this issue.  Perhaps outside the scope of this PR.

Reply:
Personally I do not think alerting the user for this is necessary since this is a very specific type of calculation. I have set up error managers for the code that depends on this attribute so I think that covers it, but open to suggestions.